api/.gitignore

```diff
@@ -0,0 +1,2 @@
+.env
+Vincent Van Bot-67b2b35a7d0c.json
```

Comment:
Even if git ignored, the name of the JSON file is still visible, which includes some sort of key. I would consider placing this in a separate folder